modules/oe_whitelabel_starter_event/oe_whitelabel_starter_event.module

```diff
@@ -171,3 +171,38 @@ function _oe_whitelabel_starter_event_preprocess_inpage_nav(array &$variables):
     ];
   }
 }
+
+/**
+ * Helper function to preprocess the date field.
+ *
+ * @param array $variables
+ *   Render array variables.
+ */
+function _oe_whitelabel_starter_event_preprocess_date(array &$variables): void {
+  /** @var \Drupal\node\NodeInterface $node */
+  $node = $variables['node'];
+
+  if ($node->get('oe_sc_event_dates')->isEmpty()) {
+    return;
+  }
+
+  $dates = $node->get('oe_sc_event_dates')->first();
+  /** @var \Drupal\Core\Datetime\DrupalDateTime $start */
+  $start = $dates->get('start_date')->getValue();
+  $variables['date_block'] = [
+    'year' => $start->format('Y'),
+    'month' => $start->format('M'),
+    'day' => $start->format('d'),
+    'date_time' => $start->format('Y-m-d'),
+  ];
+  /** @var \Drupal\Core\Datetime\DrupalDateTime $end */
+  $end = $dates->get('end_date')->getValue();
+
+  if (!empty($end) && $start->format('d-m-y') !== $end->format('d-m-y')) {
```

Comment:
Is it correct to show the year twice when it's the same?
![image](https://user-images.githubusercontent.com/8488617/211046297-eac035ac-6429-404f-b51e-823e66e5d505.png)
Maybe it's something to ask the designers.